src/app/shared/components/template/template-component.ts

```diff
@@ -42,7 +44,7 @@ export class TmplCompHostDirective {
   selector: "plh-template-component",
   template: `
     <div
-      class="plh-tmpl-comp"
+      class="plh-tmpl-comp {{ rowNameClass }}"
```

Comment:
would it not just be easier to use data attributes like below, and specify
```
[attr.data-rowname]="row.name"
```
That would also mean we don't need any of the other lines of code in this file, keeping it more concise